STM32Cube/Tasks/controller.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * controller.c
+ *
+ *  Created on: Apr 14, 2024
+ *      Author: joedo
+ */
+
+#include "controller.h"
+#include "freertos.h"
+#include "cmsis_os.h" //TODO replace with xtaskDelayUntil
+#include "stm32h7xx_hal.h"
+#include <stdio.h>
+#include <math.h>
+#include "millis.h"
+
+extern UART_HandleTypeDef huart4;
+
+
+//needed dependecy for stdio-->printf()
+int __io_putchar(int ch)
```

Comment:
should cut this since Christine's new printf lib addition deals with this

Reply:
Yup, sounds good, stdlib also not included anymore.